stable/traefik/values.yaml

```diff
@@ -28,6 +28,13 @@ acme:
   enabled: false
   email: admin@example.com
   staging: true
+  ## Set dnsProvider to perform domain verification using dns challenge
+  dnsProvider:
+    name: ""
+    config: {}
+    #   - AWS_REGION: us-east-1
+    #   - AWS_ACCESS_KEY_ID: ""
+    #   - AWS_SECRET_ACCESS_KEY: ""
```

Comment:
I thought we had discussed having a section per provider, with keys for relevant settings so that we weren't sending users straight off to the Traefik docs to figure out what do do. And then we'd need a discriminator of some sort to say which section to use.